packages/Webkul/DataTransfer/src/Console/JobExecuteCommand.php

```diff
@@ -0,0 +1,303 @@
+        });
+
+        $this->laravel['events']->listen(JobReleasedAfterException::class, function ($event) {
+            $this->writeOutput($event->job, 'released_after_exception');
+        });
+
+        $this->laravel['events']->listen(JobFailed::class, function ($event) {
+            $this->writeOutput($event->job, 'failed');
+
+            $this->logFailedJob($event);
+        });
+    }
+
+    /**
+     * Write the status output for the queue worker.
+     *
+     * @return void
+     */
+    protected function writeOutput(Job $job, string $status)
+    {
+        $this->output->write(sprintf(
+            '  <fg=gray>%s</> %s%s',
+            $this->now()->format('Y-m-d H:i:s'),
+            $job->resolveName(),
+            $this->output->isVerbose()
+            ? sprintf(' <fg=gray>%s</>', $job->getJobId())
+            : ''
+        ));
+
+        if ($status == 'starting') {
+            $this->latestStartedAt = microtime(true);
+
+            $dots = max(terminal()->width() - mb_strlen($job->resolveName()) - (
+                $this->output->isVerbose() ? (mb_strlen($job->getJobId()) + 1) : 0
+            ) - 33, 0);
+
+            $this->output->write(' '.str_repeat('<fg=gray>.</>', $dots));
+
+            return $this->output->writeln(' <fg=yellow;options=bold>RUNNING</>');
+        }
+
+        $runTime = $this->formatRunTime($this->latestStartedAt);
+
+        $dots = max(terminal()->width() - mb_strlen($job->resolveName()) - (
+            $this->output->isVerbose() ? (mb_strlen($job->getJobId()) + 1) : 0
+        ) - mb_strlen($runTime) - 31, 0);
+
+        $this->output->write(' '.str_repeat('<fg=gray>.</>', $dots));
+        $this->output->write(" <fg=gray>$runTime</>");
+
+        $this->output->writeln(match ($status) {
+            'success'                  => ' <fg=green;options=bold>DONE</>',
+            'released_after_exception' => ' <fg=yellow;options=bold>FAIL</>',
+            default                    => ' <fg=red;options=bold>FAIL</>',
+        });
+    }
+
+    /**
+     * Store a failed job event.
+     */
+    protected function logFailedJob(JobFailed $event): void
+    {
+        $this->laravel['queue.failer']->log(
+            $event->connectionName,
+            $event->job->getQueue(),
+            $event->job->getRawBody(),
+            $event->exception
+        );
+    }
+
+    /**
+     * Given a start time, format the total run time for human readability.
+     */
+    protected function formatRunTime(float $startTime): string
+    {
+        $runTime = (microtime(true) - $startTime) * 1000;
+
+        return $runTime > 1000
+            ? CarbonInterval::milliseconds($runTime)->cascade()->forHumans(short: true)
+            : number_format($runTime, 2).'ms';
+    }
+
+    /**
+     * Get the current date / time.
+     */
+    protected function now(): Carbon
+    {
+        $queueTimezone = $this->laravel['config']->get('queue.output_timezone');
+
+        if ($queueTimezone &&
```

Comment:
Fix alignment for this if condition either in multi line or in a single line.

Reply:
resolved